framework/src/utils/PetscSupport.C

```diff
@@ -878,7 +954,7 @@ setSinglePetscOption(const std::string & name,
   else if (lower_case_name.find("mat_type") != std::string::npos)
```

Comment:
this one and the one above should use the prefix for the find? Or at least vec_type instead of -vec_type?

Reply:
There is a later search for a particular match for the `mat_type`. We do not prefix the vectors anywhere currently so if the user specifies something other than `-vec_type` than their option will go unused. We can add vector prefixing, but I'd prefer to do that in follow-on work